source/initial-dns-seedlist-discovery/initial-dns-seedlist-discovery.md

```diff
@@ -200,11 +198,28 @@ mongodb://mongodb1.mongodb.com:27317,mongodb2.mongodb.com:27107/?ssl=true&replic
 
 ## Test Plan
 
+### Spec Tests 
 See README.md in the accompanying [test directory](tests).
 
 Additionally, see the `mongodb+srv` test `invalid-uris.yml` in the
 [Connection String Spec tests](../connection-string/tests).
 
+### Prose Tests
```

Comment:
I think most prose tests go in the local tests/README.md file so this change should be in `source/initial-dns-seedlist-discovery/tests/README.md`

Reply:
done!